tenable/io/v3/base/schema/explore/search.py

```diff
@@ -37,6 +57,6 @@ class SearchSchema(Schema):
     '''
     fields_ = fields.List(fields.Str(), allow_none=True, data_key='fields')
     filter = fields.Nested(FilterSchema, allow_none=True)
-    limit = fields.Int(default=1000)
+    limit = fields.Int(default=200, validate=v.Range(max=200))
```

Comment:
should we restrict the user to200 records? because we can get more records from server at once.

Reply:
Removed the validation.